src/components/App.tsx

```diff
@@ -6501,7 +6501,7 @@ class App extends React.Component<AppProps, AppState> {
             }
 
             nextElements = updateFrameMembershipOfSelectedElements(
-              this.scene.getElementsIncludingDeleted(),
+              nextElements,
```

Comment:
This was causing weird behaviour when dragging elements into the frame. Double-checked if this was some regression but was here since the beginning of #6123.

Still not 100% sure this fix is correct.